src/nbl/asset/interchange/CPLYMeshFileLoader.cpp

```diff
@@ -143,14 +145,15 @@ void CPLYMeshFileLoader::initialize()
 
 			SRasterizationParams rastarizationParmas;
 
+#if 0 // WHAT IS THIS?
```

Comment:
here

Reply:
you're supposed to compile without NBL_COMPILE_WTIH_STL_LOADER, I changed the defaults but you might have used an already exisitng cmake build-dir